api/src/test/resources/io/strimzi/api/kafka/model/KafkaConnector.out.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: "kafka.strimzi.io/v1alpha1"
+kind: "KafkaConnector"
+metadata:
+  name: "test-source"
+spec:
+  class: "org.apache.kafka.connect.file.FileStreamSourceConnector"
+  tasksMax: 2
+  topics:
+  - "test.topic"
+  config:
+  - name: "file"
+    value: "/home/source/test.source.txt"
```

Comment:
I know we discussed this before, but can't find the outcome, so sorry for asking again. Can't this be given like so instead of the name/value pairs:

    config:
     - file: "/home/source/test.source.txt"
     - otherProp: ...

Reply:
@scholzj @ppatierno do you think we should serialize as a map which is consistent with the rest of Strimzi, but not really consistent with Kubernetes API conventions? Do we think we should aim to become consistent with the Kubernetes API conventions at some point in the future?

Reply:
The name & value will give us more options to use things such as config maps and secrets if we decide so in the future (although I see it as a bit of security risk). The plain map does not five us such options.

Reply:
Agree I would stay with name-value